src/test/java/com/faforever/integration/factories/SessionFactory.java

```diff
@@ -25,14 +25,14 @@
   private static final String OAUTH_CLIENT_ID = "1234";
   private static final String OAUTH_SECRET = "secret";
 
-  private static ObjectMapper objectMapper = new ObjectMapper();
-  private static ShaPasswordEncoder shaPasswordEncoder = new ShaPasswordEncoder(256);
+  private static final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Whenever you're instantiating an `ObjectMapper` you're doing it wrong ;) inject it from outside, or add it as method parameter